apps/molgenis-components/src/components/account/MolgenisSession.vue

```diff
@@ -154,7 +154,9 @@ export default defineComponent({
       if (schemaSettings && schemaSettings._session) {
         this.session = schemaSettings._session;
       } else {
-        this.session = {};
+        this.session = {
+          settings: {},
```

Comment:
looks good but do you want to set a default ( then move to data section of options api ) or use it as a reset ( clear session in case resp does not have _session ?

Reply:
its now a reset, so it will refresh complete if user would sign in or sign out, correct? Or maybe in practice then via data() would work, I must confess I didn't try.

Reply:
ok , its fine as reset , could make it more clear by adding ```settings: {}``` to initial component setup, but it solved te described issue